From 5b904c183e00060d455009632a49a7adda789204 Mon Sep 17 00:00:00 2001
From: Charles Short <charles.short@windriver.com>
Date: Wed, 26 Jan 2022 13:32:22 +0000
Subject: [PATCH] debian: Bump timesyncd from Depends to Recommends

Allows other time sync daemons to be installed such as NTP
to be installed in order to fix the ISO build.

Signed-off-by: Charles Short <charles.short@windriver.com>
---
 debian/control | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/control b/debian/control
index be7c47b90e..c20529e354 100644
--- a/debian/control
+++ b/debian/control
@@ -65,7 +65,8 @@ Architecture: linux-any
 Multi-Arch: foreign
 Section: admin
 Priority: important
-Recommends: dbus
+Recommends: dbus,
+            systemd-timesyncd | time-daemon,
 Suggests: systemd-container,
           policykit-1
 Pre-Depends: ${shlibs:Pre-Depends},
@@ -73,7 +74,6 @@ Pre-Depends: ${shlibs:Pre-Depends},
 Depends: ${shlibs:Depends},
          ${misc:Depends},
          libsystemd0 (= ${binary:Version}),
-         systemd-timesyncd | time-daemon,
          util-linux (>= 2.27.1),
          mount (>= 2.26),
          adduser,
-- 
2.32.0

